azurerm/data_source_dns_zone.go

```diff
@@ -43,6 +43,29 @@ func dataSourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
+				Type:     schema.TypeString,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
```

Comment:
since this is a Data Source this only needs `Type: schema.TypeString` and `Computed: true` - can we remove Default/Optional?

Reply:
done.